mvn-defaults/pom.xml

```diff
@@ -549,7 +551,7 @@
             <plugin>
                 <groupId>org.jacoco</groupId>
                 <artifactId>jacoco-maven-plugin</artifactId>
-                <version>0.8.10</version>
+                <version>${jacoco.version}</version>
```

Comment:
We've moved away from indirecting these version numbers into the same place, since it makes live editing/analysis harder, and there is already a way to centralize version numbers into one place using maven dependency management.